packages/gatsby-theme-store/src/components/HomePage/BelowTheFoldPreview.tsx

```diff
@@ -1,11 +1,20 @@
 import { Center, Spinner } from '@vtex/store-ui'
-import type { FC } from 'react'
+import type { FC, CSSProperties } from 'react'
 import React from 'react'
 
 const BelowTheFoldPreview: FC = () => (
-  <Center height="500px">
-    <Spinner />
-  </Center>
+  <div
+    style={
+      ({
+        contentVisibility: 'auto',
+        containIntrinsicSize: '500px',
+      } as unknown) as CSSProperties
+    }
+  >
```

Comment:
shouldn't that be a variant?

Reply:
Hmm, so. I'm not sure. 

I was thinking this weekend and I think we should remove all theme-ui's dependencies from theme-store and leave it as a standalone package. This way we can use any styling with it. What do you think ?